sources/oracle/toddl.go

```diff
@@ -38,51 +38,91 @@ type ToDdlImpl struct {
 // mapping.  toSpannerType returns the Spanner type and a list of type
 // conversion issues encountered.
 func (tdi ToDdlImpl) ToSpannerType(conv *internal.Conv, columnType schema.Type) (ddl.Type, []internal.SchemaIssue) {
-	ty, issues := toSpannerTypeInternal(conv, columnType.Name, columnType.Mods)
+	// passing empty spType to execute default case.will get other spType from web pkg
+	ty, issues := ToSpannerTypeInternal(conv, "", columnType.Name, columnType.Mods)
 	if conv.TargetDb == constants.TargetExperimentalPostgres {
 		ty = overrideExperimentalType(columnType, ty)
 	}
 	return ty, issues
 }
 
-func toSpannerTypeInternal(conv *internal.Conv, id string, mods []int64) (ddl.Type, []internal.SchemaIssue) {
+func ToSpannerTypeInternal(conv *internal.Conv, spType string, srcType string, mods []int64) (ddl.Type, []internal.SchemaIssue) {
```

Comment:
this method is named internal, we should not expose it publicly

Reply:
We haven’t defined a different type map for oracle in the web package, we are using a mapping from toddl only as we discussed.
we need an extra argument(spanner datatype) to use exiting mapping but we can not update ToSpannerType function because it is part of the ToDdlImpl interface. So as a temporary solution we made this internal function public. Eventually will merge type maps for all the databases and update the interface function.
Should we change the function name or any better approach?